src/components/asset-list/RecyclerAssetList2/core/StickyHeaders.tsx

```diff
@@ -47,11 +48,12 @@ function StickyHeaderInternal({
   const { setMeasures, interpolationsRanges, scrollViewRef } = context;
   const { range: range_, last } = interpolationsRanges[name] || {};
   const range = range_?.map(r => r - context.yOffset);
+  const { colors } = useTheme();
 
   const position = useRecyclerAssetListPosition();
   const animatedStyle = useMemo(
     () => ({
-      backgroundColor: 'white',
+      backgroundColor: colors.white,
```

Comment:
is this ok? iirc colors.white is white in dark mode btw

Reply:
yeah this was messed up but not being used previously